discord/partial_emoji.py

```diff
@@ -25,13 +25,20 @@
 
 from __future__ import annotations
 
+import json
 import re
+from pathlib import Path
 from typing import TYPE_CHECKING, Any, TypedDict, TypeVar
 
 from . import utils
 from .asset import Asset, AssetMixin
 from .errors import InvalidArgument
 
+EMOJIS_MAP_PATH = Path(__file__).parent / "emojis.json"
```

Comment:
Please make use of this instead to ensure the file always is loaded correctly:

https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files